pkg/storage/tsdb/bucketcache/caching_bucket.go

```diff
@@ -221,7 +244,13 @@ func (cb *CachingBucket) Get(ctx context.Context, name string) (io.ReadCloser, e
 	contentKey := cachingKeyContent(name)
 	existsKey := cachingKeyExists(name)
 
-	hits := cfg.cache.Fetch(ctx, []string{contentKey, existsKey})
+	var opts []cache.Option
```

Comment:
[nit] This code is duplicated below. You may consider to move it to a function `getCacheOptions(ctx)`.